src/locales/en.json

```diff
@@ -344,8 +340,14 @@
     "pattern": "The value must match a specific pattern: %{pattern}"
   },
 
+  "update": {
+    "title": "An update is available for this service.",
+    "regular": "Perform this update from the service page to keep fetching your data and to have the latest features:",
+    "blocking": "Update it to keep fetching your data and to have the latest features:",
+    "CTA": "See the page"
```

Comment:
CTAs should be more accurate : "Update your app".

Reply:
For now, we are not redirect to the update modal (this is a next card), we are just redirected to the app page, poke @Claiw